helpers/shortcuts.ts

```diff
@@ -60,10 +60,16 @@ function transformKeys(keys: string[]) {
     if (Object.keys(keysMap).includes(key)) {
       return keysMap[key];
     }
-    return key.toUpperCase();
+    return toUpperCase(key);
   });
 }
 
+const toUpperCase = (key: string): string => {
```

Comment:
:zap: **Suggestion** 
The `toUpperCase` function is not correctly converting a lowercase letter to uppercase. The ASCII value difference between lowercase and uppercase letters is 32, not 40. This should be corrected to ensure proper functionality.